osu.Game.Rulesets.Osu/Mods/OsuModWiggle.cs

```diff
@@ -37,6 +37,11 @@ private void drawableOnApplyCustomUpdateState(DrawableHitObject drawable, ArmedS
             var osuObject = (OsuHitObject)drawable.HitObject;
             Vector2 origin = drawable.Position;
 
+            // Wiggle the repeat points with the slider instead of independently.
+            // Also fixes an issue with repeat points being positioned incorrectly.
+            if (osuObject is RepeatPoint)
```

Comment:
BTW, I did try having the check in ApplyToDrawableHitObjects, but it didn't work for some reason.

Reply:
it's probably worth investigating why